app/src/main/res/menu/fragment_account_media.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/action_refresh"
+        android:title="@string/action_refresh"
+        app:showAsAction="never" />
+</menu>
```

Comment:
Can we have 1 refresh menu instead all these duplicate files please?

Reply:
I thought about that, but I figured:

1. I plan on extending some of these to add additional menu items:

For example:

- in Notifications the "Clear" and "Filter" controls can move in to the menu
- on a timeline having a menu with "Hide boosts" and "Hide replies" (or "Show...") to temporarily override (or adjust) the preference
- when viewing edits, an option to toggle the display of what's changed

etc.

2. It makes it easier to navigate the code. If you want to see what menu an activity or fragment uses you only have to open on efile.